src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -252,6 +252,11 @@ final class SecureStore: Store {
 		get { kvStore["fakeSQLiteError"] as Int32? }
 		set { kvStore["fakeSQLiteError"] = newValue }
 	}
+	
+	var dmKillDeviceTimeCheck: Bool? {
```

Comment:
Maybe it makes sense to put default "false" into the store property?

Reply:
Didn't won't to write something into the store that only gets set if you want to debug something. Maybe I should also set it back to nil if the Toogle gets set to Off again.

Reply:
You wouldn't write it to the store. The default would come into place if the flag is read but nothing was written to the store before. At the moment you are handling the nil case with default of false at different places instead of a central one.

For example isAllowedToPerformBackgroundFakeRequests:
```
	var isAllowedToPerformBackgroundFakeRequests: Bool {
		get { kvStore["shouldPerformBackgroundFakeRequests"] as Bool? ?? false }
		set { kvStore["shouldPerformBackgroundFakeRequests"] = newValue }
	}
```

Reply:
Ah sweet, I don't know that.
Refactored :)